Jenkinsfile

```diff
@@ -44,8 +12,8 @@ pipeline {
         booleanParam(name: 'z', defaultValue: true, description: 'IBM Z (s390x) Build/Test')
         booleanParam(name: 'powerpc', defaultValue: true, description: 'PowerPC (ppc64le) Build/Test')
         booleanParam(name: 'vendor', defaultValue: true, description: 'Vendor')
-        booleanParam(name: 'windowsRS1', defaultValue: true, description: 'Windows 2016 (RS1) Build/Test')
-        booleanParam(name: 'windowsRS5', defaultValue: true, description: 'Windows 2019 (RS5) Build/Test')
+        booleanParam(name: 'windowsRS1', defaultValue: false, description: 'Windows 2016 (RS1) Build/Test')
+        booleanParam(name: 'windowsRS5', defaultValue: false, description: 'Windows 2019 (RS5) Build/Test')
```

Comment:
Does this mean both RS1 and RS5 are disabled by default?

Reply:
For now yes. The new Jenkins infra isn't set up with the windows nodes just yet.  We have to spin up windows pet instances and attach them to the ci.docker.com/public

Reply:
Yes, for now. After this PR is merged we will get more PRs to enable windows tests so the windows tests are on by default before we disable the old, hand-crafted PR checks we have now for moby/moby PRs.

Reply:
(currently we gotta get the RS1 and RS5 boxes fixed before they can be activated)

Reply:
cc @StefanScherer